server/src/main/java/umm3601/Server.java

```diff
@@ -39,6 +39,7 @@ public static void main(String[] args) {
 
     Javalin server = Javalin.create(config -> {
       config.registerPlugin(new RouteOverviewPlugin("/api"));
+      config.jsonMapper(new JavalinJackson());
```

Comment:
This can be omitted, Javalin will automatically use `new JavalinJackson()` for the JSON mapper when one isn't provided. See https://github.com/tipsy/javalin/blob/bff382795d9fb40843c6a213952fc2191c24441e/javalin/src/main/java/io/javalin/core/JavalinConfig.java#L226

```suggestion
```